packages/events-controller/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@tryghost/events-controller",
```

Comment:
This should be prefixed with `members-` as it's members specific, and probably called `@tryghost/members-analytics-ingress` or something more specific?

Reply:
Hmm yeah i was confused about the `members-` prefix because for some of the new packages we haven't used it like `@tryghost/domain-events`, so i went with that, but agree `@tryghost/members-analytics-ingress` is a much better name, will clean that up now

Reply:
Done